src/subcommand/wallet/inscribe.rs

```diff
@@ -14,14 +14,33 @@ use {
 
 #[derive(Debug, Parser)]
 pub(crate) struct Inscribe {
+  #[clap(long, help = "Inscribe on <ORDINAL>")]
   ordinal: Ordinal,
-  content: String,
+  #[clap(long, help = "Inscribe contents of <FILE>")]
+  file: PathBuf,
 }
 
 impl Inscribe {
   pub(crate) fn run(self, options: Options) -> Result {
     let client = options.bitcoin_rpc_client_mainnet_forbidden("ord wallet inscribe")?;
 
+    let inscription = match &self
+      .file
+      .extension()
+      .expect("file should always have an extension")
+      .to_str()
+    {
+      Some("txt") => Inscription::Text(String::from_utf8_lossy(&fs::read(&self.file).unwrap()).into()),
+      Some("png") => Inscription::Png(fs::read(&self.file).unwrap()),
+      _ => {
+        return Err(anyhow!("inscribe only accepts .txt and .png"));
+      }
```

Comment:
`from_utf8_lossy` replaces all invalid UTF-8 with replacement characters. Publishing that is probably not what users want.

```suggestion
      "txt" => Inscription::Text(fs::read_to_string(&self.file)?),
      "png" => Inscription::Png(fs::read(&self.file)?),
      other => bail!("Unrecognized file extension: `.{other}`"),
```

Reply:
add what extension accepted
`unrecognized file extension  `.{other}`, only .txt and .png are accepted`